py3status/py3.py

```diff
@@ -464,3 +474,16 @@ def stop_sound(self):
         if self._audio:
             self._audio.kill()
             self._audio = None
+
+    @staticmethod
```

Comment:
I'm not a fan of static methods in Py3.  My reasoning is that I might want to do some logging etc in functions without having to change things about.  It also keeps all the methods similar.

Also in this actual function I think it would be good to log the error.